resources/deu.ini

```diff
@@ -5,7 +5,7 @@
 ; Popup when entering a plot
 popup=Du hast das Grundstück {%0} §rbetreten
 popup.owner=Dieses Grundstück gehört {%0}
-popup.forsale=Dieses Plot wurde von {%0} zum Verkauf erklärt. Mit /p buy kannst du dir dieses Grundstück für {%1}$ kaufen.
+popup.forsale=Dieses Grundstück wurde von {%0} zum Verkauf erklärt. Mit /p buy kannst du dir dieses Grundstück für {%1}$ kaufen.
```

Comment:
This change is outside the scope of this PR

Reply:
Oops, you're right.